WeakAurasOptions/OptionsFrames/TextEditor.lua

```diff
@@ -522,14 +522,14 @@ local function ConstructTextEditor(frame)
       end
   )
 
-  -- CTRL + S saves and closes, ESC cancels and closes
+  -- CTRL + S saves and closes, ANYMOD + ESC cancels and closes
   editor.editBox:HookScript(
     "OnKeyDown",
     function(_, key)
       if IsControlKeyDown() and key == "S" then
         group:Close()
       end
-      if key == "ESCAPE" then
+      if IsModifierKeyDown() and key == "ESCAPE" then
```

Comment:
Just remove it, having a modifier is non intuitive and wouldn't make the ratio between actually intended and unintended usages any better.

Reply:
Fair enough. I thought it would be awkward enough to use a modifier with escape that it wouldn't happen by accident while giving people the option if they need it. But I'm certainly not going to put up a fight because I hate it :D